bindings_ffi/src/mls.rs

```diff
@@ -917,19 +917,30 @@ impl FfiConversations {
 
     pub async fn sync_all_conversations(&self) -> Result<u32, GenericError> {
```

Comment:
Are there any clients depending on the current behavior? Is it worth making a separate method called `sync_all`, or renaming this one? Don't have a strong opinion